openid-connect-server-webapp/src/main/resources/db/tables/oracle_database_tables.sql

```diff
@@ -0,0 +1,402 @@
+--
```

Comment:
Changes comparing to `hsql_database_tables.sql`:
* Oracle doesn't have a BIGINT type - changed it to NUMBER(19)
* Oracle doesn't support identity - changed it to separate sequence for every numeric ID
* Oracle doesn't have a BOOLEAN type - changed it to NUMBER(1) and added constraint check for 0 or 1 for every boolean
* some identifiers exceed Oracle's 30 character limit, so they needed to be shortened (please see `entity-mappings_oracle.xml`)
* Oracle's VARCHAR2 type doesn't support columns larger than 4000 characters, so for 4096 character columns I changed it to 4000, for 8192 I changed the datatype to CLOB